Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -147,29 +147,6 @@ namespace
         return z0;
     }
 
-    struct PDim3 {
-        ParticleReal x, y, z;
-
-        AMREX_GPU_HOST_DEVICE
-        PDim3(const amrex::XDim3& a):
-            x{static_cast<ParticleReal>(a.x)},
-            y{static_cast<ParticleReal>(a.y)},
-            z{static_cast<ParticleReal>(a.z)}
-        {}
-
-        AMREX_GPU_HOST_DEVICE
-        ~PDim3() = default;
-
-        AMREX_GPU_HOST_DEVICE
-        PDim3(PDim3 const &)            = default;
-        AMREX_GPU_HOST_DEVICE
-        PDim3& operator=(PDim3 const &) = default;
-        AMREX_GPU_HOST_DEVICE
-        PDim3(PDim3&&)                  = default;
-        AMREX_GPU_HOST_DEVICE
-        PDim3& operator=(PDim3&&)       = default;
-    };
-
```

Comment:
I had to move this object from `PhysicalParticleContainer.cpp` to `AddPlasmaUtilities.H` as part of this PR.